src/shared/i18n/index.js

```diff
@@ -0,0 +1,104 @@
+import { IntlProvider, addLocaleData } from 'react-intl';
+import { getSetting } from '../selectors';
+
+// export wrapper for file-manager.js and index.js
+export { IntlProvider };
+
+const config = {
+  standardLanguage: 'en',
+  availableLanguages: [
+    {
+      name: 'English',
+      code: 'en'
+    },
+    {
+      name: 'Deutsch',
+      code: 'de'
+    },
+    {
+      name: 'Español',
+      code: 'es'
+    }
+  ]
+};
+
+// add available language rules from react-intl
+addLocaleData(
+  config.availableLanguages.map(lang =>
+    require(`react-intl/locale-data/${lang.code}`)
+  )
+);
+
+// add all locale files to an array for electron
+const getAllTranslationsAsObject = () => {
+  const translations = [];
+
+  config.availableLanguages.forEach(lang => {
+    translations[lang.code] = require(`../../../locales/${lang.code}`);
```

Comment:
I think this wont work when compiled. Since the paths will change. in `menu.js` I do this for resolving: ```path.resolve(__dirname, `../resources/icons/${item.icon}.png`)``` and I think this should be similar.

Reply:
Also can we use `.map` here? 

``` js
const translations = config.availableLanguages.map(lang => require(...));
```

Reply:
It will compile. I've made the forEach, because if you dynamically require content, webpack typically ignores it. With this little trick I require all translations and load them into the object. So when I add a language to the config object, the translations are automatically added.
Anyways, I have updated my code and decided to import the languages by hand.